backend/PythonClient/multirotor/util/graph/three_dimensional_grapher.py

```diff
@@ -193,7 +193,11 @@ def draw_interactive_trace_vs_planned(planed_position_list, actual_position_list
             #                   scene=dict(xaxis_range=[-max_val, max_val],
             #                              yaxis_range=[-max_val, max_val],
             #                              zaxis_range=[-max_val, max_val]))
+            html_content = fig.to_html(full_html=True)
+
```

Comment:
What is the purpose of this?

Reply:
This allows me to save the contents to html format and return them for use in uploading.